airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "../../../scss/colors";
+@use "../../../scss/fonts";
+
+.circle {
+  height: 20px;
+  width: 20px;
+  min-width: 20px;
+  border-radius: 50%;
+  text-align: center;
+  margin-right: 6px;
```

Comment:
I don't think we want the `margin-right` on this component. This is meant in a place this is used to keep distance from another element. The place where this is used should apply this and not the component itself, since it does not really have any idea about the actual surroundings where this is used.

The `ConnectEntitiesCell` already sets a margin on this component anyway, in the `DiffIconBlock` we can increase the `gap` property on the parent flex box container to give more spacing between the numbers.